app.py

```diff
@@ -36,6 +36,30 @@ def list_bibliography_from_db(db, count):
     ).limit(count)
     return bibliography_sets.all()
 
+
+@app.route("/document_matrix")
+@requires_db
+async def format_term_document_matrix(db, request):
```

Comment:
This function doesn't format but "lists" the term document matrices, so what about `list_term_document_matrices(db, request)`?